.changeset/weak-rivers-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': minor
```

Comment:
This is not a breaking change right? Then this can be `patch` (since we are on 0.x versions)